docs/docs/cmd/spo/listitem/listitem-roleassignment-add.mdx

```diff
@@ -84,6 +90,12 @@ Add role assignment to specified list item in specified list located in specifie
 m365 spo listitem roleassignment add --webUrl "https://contoso.sharepoint.com/sites/project-x" --listTitle "someList" --listItemId 1 --principalId 11 --roleDefinitionName "Full Control"
 ```
 
+Add role assignment to specified list item in specified list located in specified site for specified Entra group id and specified role definition name.
+
+```sh
+m365 spo listitem roleassignment add --webUrl "https://contoso.sharepoint.com/sites/project-x" --listTitle "someList" --listItemId 1 --entraGroupId '27ae47f1-48f1-46f3-980b-d3c1470e398d' --roleDefinitionName "Full Control"
```

Comment:
its a tiny thing but we could use double quotes to align with other options defined
```suggestion
m365 spo listitem roleassignment add --webUrl "https://contoso.sharepoint.com/sites/project-x" --listTitle "someList" --listItemId 1 --entraGroupId "27ae47f1-48f1-46f3-980b-d3c1470e398d" --roleDefinitionName "Full Control"
```